test/components/views/settings/tabs/user/SessionManagerTab-test.tsx

```diff
@@ -399,5 +400,166 @@ describe('<SessionManagerTab />', () => {
             // logout dialog opened
             expect(modalSpy).toHaveBeenCalledWith(LogoutDialog, {}, undefined, false, true);
         });
+
+        describe('other devices', () => {
+            const interactiveAuthError = { httpStatus: 401, data: { flows: [{ stages: ["m.login.password"] }] } };
+
+            beforeEach(() => {
+                mockClient.deleteMultipleDevices.mockReset();
+            });
+
+            it('deletes a device when interactive auth is not required', async () => {
+                mockClient.deleteMultipleDevices.mockResolvedValue({});
+                mockClient.getDevices
+                    .mockResolvedValueOnce({ devices: [alicesDevice, alicesMobileDevice, alicesOlderMobileDevice] })
+                    // pretend it was really deleted on refresh
+                    .mockResolvedValueOnce({ devices: [alicesDevice, alicesOlderMobileDevice] });
+
+                const { getByTestId } = render(getComponent());
+
+                await act(async () => {
+                    await flushPromisesWithFakeTimers();
+                });
+
+                toggleDeviceDetails(getByTestId, alicesMobileDevice.device_id);
+
+                const deviceDetails = getByTestId(`device-detail-${alicesMobileDevice.device_id}`);
+                const signOutButton = deviceDetails.querySelector(
+                    '[data-testid="device-detail-sign-out-cta"]',
+                ) as Element;
+                fireEvent.click(signOutButton);
+
+                // sign out button is disabled with spinner
+                expect((deviceDetails.querySelector(
+                    '[data-testid="device-detail-sign-out-cta"]',
+                ) as Element)).toMatchSnapshot();
```

Comment:
Like at the other place: would be nice to check specific attributes instead of the entire snapshot.